xml/System.ComponentModel.Design/DesignerActionService.xml

```diff
@@ -603,11 +603,27 @@
         <Parameter Name="type" Type="System.Windows.Forms.Design.ComponentActionsType" Index="1" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="component">To be added.</param>
-        <param name="type">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="component">The component that the DesignerActions are associated with.</param>
+        <param name="type">The <see cref="T:System.ComponentModel.Design.ComponentActionsType" /> to filter the associated designer actions with.</param>
+        <summary>Returns the collection of designer action item lists of the specified type associated with a component.</summary>
+        <returns>The collection of designer actions of the specified type for the specified component.</returns>
+        <exception cref="T:System.ArgumentNullException">If the component parameter is null ArgumentNullException is thrown.</exception>
```

Comment:
These descriptions for the exceptions can be better. Use this one as an example for the other updated areas. Don't reiterate the exception type, as that is indicated by the `cref` parameter. Feel free though to use the `<paramref>` syntax when identifying the parameter causing the exception.

```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="component" /> is <see langword="null" />.</exception>
```